src/components/ToggleSwitch/index.tsx

```diff
@@ -80,13 +81,13 @@ const styles = StyleSheet.create({
   },
   labelLeft: {
     marginRight: dimen.normalizeDimen(5),
-    color: '#FFFFFF',
+    color: COLORS.white,
     fontSize: normalizeFontSize(12),
     fontWeight: '400'
   },
   labelRight: {
     marginLeft: dimen.normalizeDimen(5),
-    color: '#FFFFFF',
+    color: COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These changes replace hard-coded color values with dynamic ones from a theme object, which is a good practice for maintaining consistency and modularity in the code.
```diff
-    color: '#FFFFFF',
+    color: COLORS.white,
-    color: '#FFFFFF',
+    color: COLORS.white,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->